### DISCUSSION TITLE
Cosmetic for Donation Shop:  Warlock green fire.

### DISCUSSION THREAD

**Discussion by Vyperhand** [chromiecraft/chromiecraft#5515] [Suggestions]
As the title how about some green fire for warlocks? This is purely cosmetic and is not attainable in-game because the quest hadn't been introduced yet at this point.  If this is something doable, that would make a very nice donation shop reward.

Cheers. :)

**Comment by cachocarne** [comment]
I believe there are addons for that that only affect the client side. Check it here:
https://www.youtube.com/watch?v=pddaIx62h4k
Link (cannot guarantee it's safe): https://www.mediafire.com/file/oc75ngl74xm3whr/Fel_Fire_Patch_V_1.7.zip
